requirements.txt

```diff
@@ -1,6 +1,8 @@
 # These are the packages that need to be installed in order to build HTML
 # documentation of the User's Guide.
 
+git-lfs
+natcap.invest
```

Comment:
Hey, I know this PR is still in draft, but I noticed this line and thought this might be useful: since the user's guide is intended to be a continuously-updated resource and we're now building parts of the UG off of the state of InVEST, what would you think about installing `natcap.invest` from the latest git source ([pip docs](https://pip.pypa.io/en/stable/cli/pip_install/#git))?